src/commcare_cloud/fab/operations/release.py

```diff
@@ -205,7 +205,6 @@ def create_code_dir(full_cluster=True):
     roles_to_use = _get_roles(full_cluster)
 
     @roles(roles_to_use)
-    @parallel
```

Comment:
This was causing a fatal issue during first deployment. Removing the decorator and running it sequentially solved it.

Reply:
Eish, ok. It might be worth looking into then.
If I remember correctly the issue was related to an ambiguous input or something which resulted from the parallel execution.
I think I'll spend a bit of time seeing how to get around this, even if it means using a different command like you mentioned.